shared/js/icons_helper.js

```diff
@@ -48,7 +48,16 @@
     iconTargetSize = iconTargetSize * window.devicePixelRatio;
 
     // First look for an icon in the manifest.
-    if (siteObj.webManifestUrl && siteObj.webManifest) {
+    if (siteObj.manifest) {
```

Comment:
Shouldn't we prioritise the web manifest spec that is standardised over the open web app manifest that is proprietary?
First, check for icons in web manifest, if none, fallback to open web app manifest.
cc @benfrancis